src/main/java/de/qabel/desktop/inject/config/StaticRuntimeConfiguration.java

```diff
@@ -49,4 +55,45 @@ public Stage getPrimaryStage() {
     public ClientDatabase getConfigDatabase() {
         return configDatabase;
     }
+
+    public void setWindow (Pane window) {
+        this.window = window;
+    }
+
+    public Pane getWindow() {
+        return window;
+    }
+
+    public void loadAboutFiles() {
+        loadThanksFile("/files/thanks_file");
+    }
+
+    private void loadThanksFile(String thanksFilePath) {
+        try {
+            thanksFileContent = readFile(thanksFilePath);
+        } catch (IOException e) {
+            showErrorAlertLoadFiles();
+        } catch (NullPointerException ignored) {
+        }
+    }
+
+    private String readFile(String filePath) throws IOException {
+        try (InputStream thanksFile = System.class.getResourceAsStream(filePath)) {
+            return IOUtils.toString(thanksFile, "UTF-8");
+        } catch (NullPointerException ignored) {
+        }
+        return null;
```

Comment:
we shouldn't introduce nulls. Either wrap a ValueObject or use an empty String. ValueObject would have the advantage that you can easily check if the file loading was successful (e.g. for the button to be disabled) but you wouldn't lead the danger that anyone checks the content and gets a null.